cmd/serverless/api_key.go

```diff
@@ -147,3 +148,17 @@ func extractRegionFromSecretsManagerArn(secretsManagerArn string) (string, error
 
 	return arnObject.Region, nil
 }
+
+func sendAPIKeyToShell(apiKey string, hostAndPort string) bool {
+	// TODO:maxday remove when ready for GA
```

Comment:
Why not just remove experimental from the name, and leave it in? Also should check it's value is set to true. Eg, if I set DD_EXPERIMENTAL_SHELL_ENABLED=false, that would enable the feature.